### ISSUE TITLE
[Order Creation] Accessibility → The Show Billing button is not accessible

### ISSUE THREAD

**Issue by shiki** [woocommerce/woocommerce-android#5820]
The Show Billing button is not accessible using TalkBack. In this video, I am navigating via element (swipe right), and TalkBack just skips the Show Billing button. 

https://user-images.githubusercontent.com/198826/152885599-5085c8cc-5642-4783-8b37-c6b4cfa382d3.mp4
